typescript/infra/src/core/govern.ts

```diff
@@ -42,39 +49,91 @@ export class AbacusCoreGovernor<Chain extends ChainName> {
   }
 
   logCalls() {
-    objMap(this.calls, (chain, calls) => {
-      console.log(chain, calls);
+    const logFn = async (
+      chain: Chain,
+      _: ChainConnection,
+      calls: types.CallData[],
+    ) => console.log(chain, calls);
+    return this.executeCalls(this.connectionFn, logFn);
+  }
+
+  protected async executeCalls(
```

Comment:
maybe these should be on `ChainConnection`?

Reply:
Good suggestion